people/people.yml

```diff
@@ -1394,18 +1406,22 @@
   description: Acoustic Species ID since 2022
   level: Project Candidate
   start: 2022
+  end: 2023
 - name: Joe Yan
   description: Acoustic Species ID since 2022
   level: Project Member
```

Comment:
Alumni

Reply:
Student list hasn't previously had "Alumni" label added to Project Candidates who have left E4E. Do we want to update all previous candidates to specify Alumni? (Do we even need to list candidates on our website?)

Reply:
Project Candidates don't show up on the website.  I guess if they've left without becoming a project member, we just drop them from the list - don't qualify them as alumni.